### PR TITLE
TASK: Add new screenshots

### PR DESCRIPTION
**Pull request by crydotsnake**
Hello everybody 👋🏼 

This pull request contains the new screenshots, for the new django-filer version 2.1.1

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,11 +17,11 @@ handling of files and images a breeze.
    pypi. github would render relative paths correctly.
 
 +--------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------+
-| .. image:: https://raw.githubusercontent.com/divio/django-filer/master/docs/_static/filer_2.png        | .. image:: https://raw.githubusercontent.com/divio/django-filer/master/docs/_static/filer_3.png        |
+| .. image:: https://raw.githubusercontent.com/django-cms/django-filer/master/docs/_static/filer_2.png        | .. image:: https://raw.githubusercontent.com/django-cms/django-filer/master/docs/_static/filer_3.png        |
```
Comment (marksweb) [inline README.rst:20]: The table markup is broken due to the different length of these URLs now.

It's a problem in the middle & at the end. Hopefully this screenshot illustrates what I'm talking about;
Screenshot 2021-11-10 at 23 56 46
w.

**Review by marksweb — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (crydotsnake) [reply README.rst:20]: In VSCode it looks like that:

Bildschirmfoto 2021-11-11 um 14 50 56

Comment (marksweb) [reply README.rst:20]: Yeah the `+` needs to lineup with the pipes (`|`) to be valid & pass the checks I think

**Comment by crydotsnake** [comment]
Every checks are green now ! :) @jrief @marksweb

**Review by crydotsnake — COMMENTED** [review]
Tables are fixed! :)

**Review by jrief — APPROVED** [review]

**Pull request merged**